api/inventory/json/client/services/add_mongo_d_b_service_responses.go

```diff
@@ -110,6 +155,44 @@ func (o *AddMongoDBServiceBody) UnmarshalBinary(b []byte) error {
 	return nil
 }
 
+/*AddMongoDBServiceDefaultBody ErrorResponse is a message returned on HTTP error.
+swagger:model AddMongoDBServiceDefaultBody
+*/
+type AddMongoDBServiceDefaultBody struct {
+
+	// code
+	Code int32 `json:"code,omitempty"`
+
+	// error
+	Error string `json:"error,omitempty"`
+
+	// message
+	Message string `json:"message,omitempty"`
+}
+
+// Validate validates this add mongo d b service default body
+func (o *AddMongoDBServiceDefaultBody) Validate(formats strfmt.Registry) error {
+	return nil
+}
+
+// MarshalBinary interface implementation
+func (o *AddMongoDBServiceDefaultBody) MarshalBinary() ([]byte, error) {
+	if o == nil {
+		return nil, nil
+	}
+	return swag.WriteJSON(o)
+}
+
+// UnmarshalBinary interface implementation
+func (o *AddMongoDBServiceDefaultBody) UnmarshalBinary(b []byte) error {
+	var res AddMongoDBServiceDefaultBody
+	if err := swag.ReadJSON(b, &res); err != nil {
+		return err
+	}
+	*o = res
+	return nil
+}
+
 /*AddMongoDBServiceOKBody add mongo d b service o k body
```

Comment:
Try to add `--additional-initialism=ok` and `--additional-initialism=db`.

Reply:
ok